pkg/controller/vlan/bridge_vlan_controller.go

```diff
@@ -82,8 +82,14 @@ func (c *BridgeVLANController) OnChange(key string, setting *harvesterv1.Setting
 		return nil, fmt.Errorf("encode network settings failed, error: %w, networksettings: %+v", err, networkSetting)
 	}
 
-	harvesterv1.SettingConfigured.True(settingCopy)
-	harvesterv1.SettingConfigured.Reason(settingCopy, "")
+	if networkSetting.NIC == "" {
+		harvesterv1.SettingConfigured.False(settingCopy)
+		harvesterv1.SettingConfigured.Reason(settingCopy, "")
```

Comment:
Why not put this line out of if-else block?

Reply:
updated